File: WorldFeed/Server/src/WorldFeed.Common/Public/Models/Interfaces/DTO/ISearchResultsDTO.cs
namespace WorldFeed.Common.Public.Models.Interfaces.DTO
{
    using WorldFeed.Common.DTO;

    public interface ISearchResultsDTO
    {
        TweetWithSearchMetadataDTO[] TweetDTOs { get; set; }

        ISearchMetadata SearchMetadata { get; set; }
    }
}

File: Sharebook/Client/src/app/core/Public/Models/Interfaces/DTO/ISearchResultsDTO.ts
import {TweetWithSearchMetadataDTO} from "../../../../Core/DTO/TweetWithSearchMetadataDTO";
import {ISearchMetadata} from "./ISearchMetadata";

export interface ISearchResultsDTO {
  tweetDTOs: TweetWithSearchMetadataDTO[];
  searchMetadata: ISearchMetadata;
}
